.travis.yml

```diff
@@ -6,12 +6,10 @@ git:
 matrix:
     fast_finish: true
     include:
-        - php: 7.2
-          env:
-            - COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
-            - COVERAGE=1
-        - php: 7.3
         - php: 7.4
```

Comment:
There should be a `7.4` entry without `--prefer-stable --prefer-lowest`

Reply:
This can now be completely scrapped, since we have GitHub actions covering all cases

Reply:
Only case missing would be the nightly build for php8 as mentioned in #208 but this should not be considered here in this patch. I will rebase this today 👍